README

```diff
@@ -67,7 +67,6 @@ Ctrl+Alt+Equal = Font size up 1
 Ctrl+Alt+Minus = Font size down 1
 Ctrl+Alt+0 = Reset font size
 Ctrl+Alt+9 = Big font size
-Ctrl+Alt+t = Open new terminal tab
 Ctrl+t = Set terminal title
```

Comment:
Why is it still wrong.  Binding for new tab is ctrl shift t, and set title
is ctrl t
On Jul 3, 2016 9:45 AM, "Boris Faure" <notifications@github.com> wrote:

> In README
> <https://github.com/billiob/terminology/pull/51#discussion_r69391922>:
>
> > @@ -67,7 +67,6 @@ Ctrl+Alt+Equal = Font size up 1
> >  Ctrl+Alt+Minus = Font size down 1
> >  Ctrl+Alt+0 = Reset font size
> >  Ctrl+Alt+9 = Big font size
> > -Ctrl+Alt+t = Open new terminal tab
> >  Ctrl+t = Set terminal title
>
> This is still wrong
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/billiob/terminology/pull/51/files/1889c6b06fa67704594fd55a1635e15ed8764139..6d53256000123e692db02bde83ba1d90dc9060b5#r69391922>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AH0qovx5q6SUvFboL519i_AO2EDMEFTBks5qR7zpgaJpZM4I89zT>
> .
>